build/Microsoft.DotNet.Cli.Compile.targets

```diff
@@ -29,7 +29,7 @@
 
     <ItemGroup>
       <CompileStageInputs Include="$(SrcDirectory)/**/*.cs" Exclude="$(SrcDirectory)/**/obj/**/*.cs" />
-      <CompileStageInputs Include="$(SrcDirectory)/**/project.json;$(SrcDirectory)/**/project.lock.json" />
+      <CompileStageInputs Include="$(SrcDirectory)/**/*.csproj;$(SrcDirectory)/**/project.assets.json" />
```

Comment:
@nguerrera should we be doing our own incrementality atop csproj? What does a sln file do in this scenario? I assume it just invokes build on the csproj and counts on fast exit from the csproj file...